package-lock.json

```diff
@@ -6,7 +6,7 @@
   "packages": {
     "": {
       "name": "linkfree",
-      "version": "1.12.0",
+      "version": "1.24.9",
```

Comment:
We normally do not commit changes into `package-lock.json`. Could you revert the changes made in this file?

Reply:
Yeah. Sure. Sorry.